### ISSUE TITLE
Lazy Module - Stimulsoft Angular  Viewer

### ISSUE THREAD

**Issue by edwinn1990** [stimulsoft/Samples-Angular#1]
Hi there 

There is an issue when importing StimulsoftViewerModule into the child module or Shared Module. 

Please refer error message below
 "BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead."

Seems like library for stimulsoft angular is importing the BrowserModule with it thus the above issue . 

Let me know workarounds on how to use Stimulsoft in the child modules 

Thanks

**Comment by VadimMatveev** [comment]
Hello.

BrowserModule is not import-ed in stimulsoft-viewer-angular component
Is sample app.module.ts used to run application, so you don't need it in your npm package, you need to add
import { StimulsoftViewerModule } from 'stimulsoft-viewer-angular';
in your root module.

**Comment by edwinn1990** [comment]
Hi 

Refer here for sample project that replicates this issue https://drive.google.com/drive/folders/1zQL8oJdxkFL5pUJquA38UeMNmhBNQTew?usp=sharing

Idea is to have reports defined in a separate module other than the app module

**Comment by ADMIADev** [comment]
Yes im with edwinn1990, it would be very nice to be able to use it in separate modules.

**Comment by VadimMatveev** [comment]
Hello @edwinn1990 

Your link is inaccessible

**Comment by edwinn1990** [comment]
> Hello @edwinn1990
> 
> Your link is inaccessible

Please try now ..

**Comment by edwinn1990** [comment]
> Hi
> 
> Refer here for sample project that replicates this issue https://drive.google.com/drive/folders/1zQL8oJdxkFL5pUJquA38UeMNmhBNQTew?usp=sharing
> 
> Idea is to have reports defined in a separate module other than the app module

Hi there 

Any update for this issue 

Thanks

**Comment by VadimMatveev** [comment]
Hello, @edwinn1990 

We need time to prepare answer for you

**Comment by VadimMatveev** [comment]
Hello @edwinn1990 

We made improvements, please do like in samples:
add imports HttpClientModule, BrowserAnimationsModule, FormsModule to app.module

**Comment by edwinn1990** [comment]
Hi 

Can you modify the sample project i sent and send it back  . Please note we would like to define reports in the separate module other then app module ..

Thanks

**Comment by VadimMatveev** [comment]
Hello.

Here you are  https://drive.google.com/file/d/1gedF2ppUUJa8yUH6nJWJ-7Ms3OpKD9au/view?usp=sharing
